tianshou/policy/imitation/cql.py

```diff
@@ -167,6 +177,23 @@ def calc_random_values(
 
         return random_value1 - random_log_prob1, random_value2 - random_log_prob2
 
+    @override
```

Comment:
override or overrides?

Reply:
overrides is the package, the annotation is override (also in python 3.12), so it's correct